pkg/cli/admin/upgrade/status/controlplane.go

```diff
@@ -429,21 +439,53 @@ func vagueUnder(actual, estimated time.Duration) string {
 
 var controlPlaneStatusTemplate = template.Must(
 	template.New("controlPlaneStatus").
-		Funcs(template.FuncMap{"shortDuration": shortDuration, "vagueUnder": vagueUnder}).
+		Funcs(template.FuncMap{"shortDuration": shortDuration, "vagueUnder": vagueUnder,
+			"updatingLine": func(elems []UpdatingClusterOperator) string {
+				if len(elems) == 0 {
+					return ""
+				}
+				var messages []string
+				for _, e := range elems {
+					message := e.Name
+					if e.Condition.Reason != "" {
+						message = fmt.Sprintf("%s: %s", e.Name, e.Condition.Reason)
+					}
+					messages = append(messages, message)
+				}
+				return fmt.Sprintf("Updating:        %s\n", strings.Join(messages, ", "))
+			},
+		}).
```

Comment:
Can we extract the function to a standard normal named one, just like the other two are? Separating the levels of abstraction makes both places easier to understand.

Reply:
Done.